packages/codex-context/src/backend.ts

```
@@ -206,9 +209,12 @@
 
 export class OllamaOpenAIBackend implements BackendClient {
-  private log = createLogger("codex-context", {
-    component: "backend",
-    driver: "ollama-openai",
+  private log = createLogger({
+    service: "codex-context",
+    base: {
+      component: "backend",
+      driver: "ollama-openai",
+    },
   });
```

ESLint

[error] Property should have a readonly modifier.

(functional/prefer-immutable-types)